back/spec/acceptance/projects_spec.rb

```diff
@@ -534,6 +539,11 @@
         expect(json_response.dig(:data, :relationships, :areas, :data).size).to eq 0
       end
 
+      example 'Clear all topics', document: false do
+        do_request(project: { topic_ids: [] })
+        expect(json_response.dig(:data, :relationships, :topics, :data).size).to eq 0
```

Comment:
Isn't it 0 without `do_request(project: { topic_ids: [] })` too?

Reply:
I still don't understand how topics are tied to projects in this test (project is created without any topics here). And so, why they're removed. Do you know how?